napalm/base.py

```diff
@@ -364,3 +364,44 @@ def get_interfaces_counters(self):
             }
         """
         raise NotImplementedError
+
+        def show_route(self, destination = ''):
+            """
+            Returns a dictionary of dictionaries containing details of all available routes to a destination.
+            Each inner dictionary contains the following fields:
+                * active_route (boolean)
+                * age (int),
+                * as_path (string)
+                * local_preference (int)
+                * next_hop (str)
+                * outgoing_interface (str)
+                * preference (int)
+                * protocol (int)
+
+            Example:
+                {
+                    "1.0.0.0/24": [
+                        {
+                            "active_route"      : True,
```

Comment:
I guess you are doing this because you plan to return all possible routes, are you contemplating ECMP routes as well? Would I get multiple routes with this attribute set to True?